clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -498,6 +499,20 @@ def _is_unschedulable(self, pod: KubernetesPod) -> bool:
                 return True
         return False
 
+    def get_node_priority(self, node_id: str) -> float:
+        pods = self._pods_by_ip[node_id]
+        if not pods or len(pods) == 0:
```

Comment:
i'm pretty sure `not pods` covers both nil and empty so you could omit `or len(pods)`